frontend/src/pages/LogsPage/LogsPage.tsx

```diff
@@ -166,8 +167,9 @@ const LogsPageInner = ({ timeMode, logCursor, presetDefault }: Props) => {
 
 	const { projectId } = useNumericProjectId()
 	const { data: histogramData, loading: histogramLoading } =
-		useGetLogsHistogramQuery({
+		useGetMetricsQuery({
```

Comment:
the logs histogram looks really spread out on my machine. maybe we could make each bar fill more space? or we could keep it more consistent with the traces page by adding a second graph on the right (maybe showing number of logs by service name?) so that each one has less horizontal space 

![Screenshot from 2024-06-07 08-35-33](https://github.com/highlight/highlight/assets/1351531/e6e21eb8-8670-42c9-99df-07a3be4e2740)

Reply:
right now, the bars have a max width (same w/ the metrics graphs) and above that the gap between bars gets larger. Agree that in this case with a small height and large width it looks funny. Another option is to query for more buckets? Will bring this up in demos

Reply:
updated to 90, looks better imo
![Screen Shot 2024-06-07 at 9 25 17 AM](https://github.com/highlight/highlight/assets/86132398/7640b562-3232-4583-880e-91ebcd72f35d)